tools/yulPhaser/GeneticAlgorithms.cpp

```diff
@@ -48,3 +50,28 @@ void RandomAlgorithm::runNextRound()
 			m_options.maxChromosomeLength
 		);
 }
+
+void GenerationalElitistWithExclusivePools::runNextRound()
+{
+	double elitePoolSize = 1.0 - (m_options.mutationPoolSize + m_options.crossoverPoolSize);
+	RangeSelection elite(0.0, elitePoolSize);
+
+	m_population =
+		m_population.select(elite) +
+		m_population.select(elite).mutate(
```

Comment:
Is it good to only mutate the elite?

Reply:
Ah, "elite" is everything that is left over - not just 10% or something, right?

Reply:
Yes, elite is not touched.

This expression creates three distinct populations and then adds them together to get the final result:
1. the elite (just selection, without modifications)
2. mutation pool (mutated elite)
3. crossover pool (crossed elite)